packages/flutter/test/material/bottom_sheet_test.dart

```diff
@@ -1794,6 +1795,39 @@ void main() {
     },
   );
 
+  testWidgets(
+    'The framework removes all animation listeners from foreign controllers when disposing.',
+    (WidgetTester tester) async {
+      final GlobalKey<ScaffoldState> scaffoldKey = GlobalKey<ScaffoldState>();
+      final _StatusTestAnimationController controller = _StatusTestAnimationController(
```

Comment:
We need to dispose of this controller once the test is done to avoid test leaks. So could you add an `addTearDown` disposing of it?